tools/create_service_bundle.py

```diff
@@ -0,0 +1,568 @@
+
+    def create_stats_file(self, task_id):
+        command = "curl -s ${MESOS_CONTAINER_IP}:${PORT_HTTP}/_stats"
+        rc, stdout, stderr = self.task_exec(task_id, command)
+        self.write_file("elasticsearch_stats_{}.json".format(task_id), stdout)
+
+    def create_tasks_stats_files(self):
+        self.for_each_running_task_with_prefix("master", self.create_stats_file)
+
+    def create(self):
+        logger.info("Creating Elastic bundle")
+        self.create_tasks_stats_files()
+
+
+class HdfsBundle(BaseTechBundle):
+    def create(self):
+        logger.info("Creating HDFS bundle")
+
+
+class KafkaBundle(BaseTechBundle):
+    def create(self):
+        logger.info("Creating Kafka bundle")
+
+
+BASE_TECH_BUNDLE = {
+    "beta-cassandra": CassandraBundle,
+    "beta-elastic": ElasticBundle,
+    "beta-hdfs": HdfsBundle,
+    "beta-kafka": KafkaBundle,
+    "cassandra": CassandraBundle,
+    "elastic": ElasticBundle,
+    "hdfs": HdfsBundle,
+    "kafka": KafkaBundle,
+}
+
+
+class ServiceBundle(Bundle):
+    DOWNLOAD_FILES_WITH_PATTERNS = ["^stdout(\.\d+)?$", "^stderr(\.\d+)?$"]
+
+    @retrying.retry(
+        wait_fixed=DEFAULT_RETRY_WAIT,
+        stop_max_attempt_number=DEFAULT_RETRY_MAX_ATTEMPTS,
+    )
+    def install_service_cli(self):
+        sdk_cmd.run_cli(
+            "package install {} --cli --yes".format(self.package_name),
+            print_output=False,
+        )
+
+    @retrying.retry(
+        wait_fixed=DEFAULT_RETRY_WAIT,
+        stop_max_attempt_number=DEFAULT_RETRY_MAX_ATTEMPTS,
+    )
+    def create_dcos_version_file(self):
+        output = sdk_cmd.run_cli("--version", print_output=False)
+        self.write_file("dcos_version.txt", output)
+
+    @retrying.retry(
+        wait_fixed=DEFAULT_RETRY_WAIT,
+        stop_max_attempt_number=DEFAULT_RETRY_MAX_ATTEMPTS,
+    )
+    def create_task_file(self):
+        output = sdk_cmd.run_cli("task --json", print_output=False)
+        self.write_file("dcos_task.json", output)
+
+    @retrying.retry(
+        wait_fixed=DEFAULT_RETRY_WAIT,
+        stop_max_attempt_number=DEFAULT_RETRY_MAX_ATTEMPTS,
+    )
+    def create_marathon_task_list_file(self):
+        output = sdk_cmd.run_cli("marathon task list --json", print_output=False)
+        self.write_file("dcos_marathon_task_list.json", output)
+
+    @retrying.retry(
+        wait_fixed=DEFAULT_RETRY_WAIT,
+        stop_max_attempt_number=DEFAULT_RETRY_MAX_ATTEMPTS,
+    )
+    def create_configuration_file(self):
+        output = sdk_cmd.svc_cli(
+            self.package_name, self.service_name, "describe", print_output=False
+        )
+
+        self.write_file("service_configuration.json", output)
+
+    @retrying.retry(
+        wait_fixed=DEFAULT_RETRY_WAIT,
+        stop_max_attempt_number=DEFAULT_RETRY_MAX_ATTEMPTS,
+    )
+    def create_pod_status_file(self):
+        output = sdk_cmd.svc_cli(
+            self.package_name,
+            self.service_name,
+            "pod status --json",
```

Comment:
Does this show the status of all pods? What about `pod info` or is that just an alias?

Reply:
Yes. `pod info` only works for a particular pod and doesn't output the full pod state (non-JSON output).